frontend/appflowy_flutter/lib/window/app_window.dart

```diff
@@ -21,12 +25,21 @@ class AppWindow {
     }
 
     await windowManager.ensureInitialized();
+    final prefs = await SharedPreferences.getInstance();
+    final windowSize = json.decode(
+      prefs.getString('windowSize') ?? '{"height": 600.0, "width": 800.0}',
+    );
```

Comment:
I suggest making a new class to handle the logic for setting and getting.

for example.

```dart
class WindowSizeManager {
  ...

  void saveSize()
  // ...

  void getSize() 
  // ...
}
```

Reply:
and move the magic number to the new class.